apps/meteor/server/startup/migrations/v270.ts

```diff
@@ -0,0 +1,21 @@
+import { VoipRoom } from '@rocket.chat/models';
+
+import { addMigration } from '../../lib/migrations';
+
+addMigration({
+	version: 270,
+	async up() {
+		// mark all voip rooms as inbound which doesn't have any direction property set or has an invalid value
+		await VoipRoom.updateMany(
+			{
+				t: 'v',
```

Comment:
I realised today that this migration query was not very specific, which was resulting in adding the direction prop to all rooms (even non-omnichannel related). This new query should fix that issue